examples/time_series_anomaly_demo.py

```
@@ -0,0 +1,51 @@
+        signal[idx] += 2.5 * np.random.choice([-1, 1])  # Add large positive or negative spike
+    return t, signal
+
+
+def main() -> None:
+    """Run the anomaly detection demo."""
+    try:
+        import matplotlib.pyplot as plt
+    except ImportError:
+        print(
+            "Error: matplotlib is required for visualization. "
+            "Install with: pip install matplotlib"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)